tests/unit/storageExplorer/models/DirOperations.spec.ts

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { Dirent, ReadStream } from 'fs';
+import { BadRequestError, NotFoundError } from '@map-colonies/error-types';
+import { DirOperations } from '../../../../src/common/utilities';
+import { fileStreamSnap, generateRootDirSnap } from '../snapshots';
+import { streamToString } from '../utils';
```

Comment:
imports order first "../../kuku" then "../muku"
just to be consistent with **prettier**

Reply:
fixed? my prettier was happy either way